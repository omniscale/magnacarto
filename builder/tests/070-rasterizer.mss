#hillshade {
  raster-opacity: 1;
  raster-scaling: lanczos;
  raster-comp-op: grain-merge;
}
#slope {
  raster-comp-op: grain-extract;
  raster-opacity: 1;
  raster-scaling: lanczos;
  raster-colorizer-default-mode: linear;
  raster-colorizer-default-color: transparent;
  raster-colorizer-stops:
    stop(0,#fff)
    stop(90, #000)
}
#dem {
  raster-opacity: 1;
  raster-scaling: lanczos;
  raster-colorizer-default-mode: linear;
  raster-colorizer-default-color: transparent;
  raster-colorizer-stops:
    stop(0,#47443e)
    stop(50, #77654a)
    stop(100, rgb(85,107,50))
    stop(200, rgb(187, 187, 120))
    stop(255, rgb(217,222,170));
  raster-comp-op: color-dodge;
}