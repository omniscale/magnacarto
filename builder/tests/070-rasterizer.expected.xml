<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="hillshade" filter-mode="first">
    <Rule>
      <RasterSymbolizer comp-op="grain-merge" opacity="1" scaling="lanczos"></RasterSymbolizer>
    </Rule>
  </Style>
  <Style name="slope" filter-mode="first">
    <Rule>
      <RasterSymbolizer comp-op="grain-extract" default-color="rgba(0, 0, 0, 0.00000)" default-mode="linear" opacity="1" scaling="lanczos">
        <stop value="0" color="#ffffff"></stop>
        <stop value="90" color="#000000"></stop>
      </RasterSymbolizer>
    </Rule>
  </Style>
  <Style name="dem" filter-mode="first">
    <Rule>
      <RasterSymbolizer comp-op="color-dodge" default-color="rgba(0, 0, 0, 0.00000)" default-mode="linear" opacity="1" scaling="lanczos">
        <stop value="0" color="#47443e"></stop>
        <stop value="50" color="#77654a"></stop>
        <stop value="100" color="#556b32"></stop>
        <stop value="200" color="#bbbb78"></stop>
        <stop value="255" color="#d9deaa"></stop>
      </RasterSymbolizer>
    </Rule>
  </Style>
  <Layer name="hillshade" srs="" status="off">
    <StyleName>hillshade</StyleName>
  </Layer>
  <Layer name="slope" srs="" status="off">
    <StyleName>slope</StyleName>
  </Layer>
  <Layer name="dem" srs="" status="off">
    <StyleName>dem</StyleName>
  </Layer>
</Map>